prow/plugins/config.go

```diff
@@ -1464,8 +1473,27 @@ func ResolveBugzillaOptions(parent, child BugzillaBranchOptions) BugzillaBranchO
 			output.DependentBugStatuses = parent.DependentBugStatuses
 			output.DependentBugStates = mergeStatusesIntoStates(output.DependentBugStates, parent.DependentBugStatuses)
 		}
-		if parent.DependentBugTargetRelease != nil {
-			output.DependentBugTargetRelease = parent.DependentBugTargetRelease
+		if parent.DependentBugTargetReleases != nil {
+			output.DependentBugTargetReleases = parent.DependentBugTargetReleases
+		}
+		if parent.DeprecatedDependentBugTargetRelease != nil {
```

Comment:
This does not seem DRY

Reply:
> This does not seem DRY

Well, I'd really like `ResolveBugzillaOptions` to be private for the config repo.  Then I'd rework the callers to ensure that we were always chaining these down to a dummy root.  And then we'd only need to perform unification logic like this in the child (and parents would be unified when recursion had them as the child).  But that seemed like more than I should bite off in this PR ;)